### PR TITLE
rails version tests

### PR DESCRIPTION
**Pull request by jdoconnor**
@lumoslabs/platform

### CODE REVIEW

**Comment by jdoconnor** [comment]
no functional changes.... just a looser gemspec.

File: .travis.yml [hunk 1]
```diff
@@ -3,9 +3,22 @@ notifications:
...
   - RAILS_ENV=test bundle exec rake konacha:run
   - cd ../..
   - bundle exec rspec
+
+gemfile:
+  - Gemfile
+  - Gemfile-rails-4.1.x
+  - Gemfile-rails-4.2.x
+  - Gemfile-rails-5.0.x
+  - Gemfile-rails-5.1.x
```
Comment (zvkemp) [inline .travis.yml:19]: This file doesn't exist

Comment (jdoconnor) [reply .travis.yml:19]: thanks... adding it now. Its an expected failure anyways.

**Comment by jdoconnor** [comment]
do you think this deserves a version bump?

**Comment by zvkemp** [comment]
Version bump is probably necessary (not sure rubygems would let you push up dependency changes otherwise)

**Pull request merged**